thorlcr/graph/thgraphslave.cpp

```diff
@@ -1856,13 +1821,10 @@ class CLazyFileIO : public CInterface, implements IFileIO, implements IDelayedFi
             return cycle_to_nanosec(getStatistic(StCycleDiskWriteIOCycles));
         }
 
-        CriticalBlock b(crit);
-        unsigned __int64 openValue = iFileIO ? iFileIO->getStatistic(kind) : 0;
+        Owned<IFileIO> _iFileIO = getFileIO();
+        unsigned __int64 openValue = _iFileIO ? iFileIO->getStatistic(kind) : 0;
```

Comment:
One reason I'm not keen on it - the second use should be _iFileIO, but too similar to distinguish.